Description: Add missing includes to ijar in order to support gcc11 build
Origin: https://gitlab.alpinelinux.org/alpine/aports/-/blob/2efd9732b4cd406ea424e5af71c915092a801e8c/testing/bazel4/patch_ftbfs_gcc11_1.patch
Acked-by: Jose Luis Rivero <jrivero@openrobotics.org>

diff --git a/third_party/ijar/mapped_file_unix.cc b/third_party/ijar/mapped_file_unix.cc
index 6e3a908..65179e3 100644
--- a/third_party/ijar/mapped_file_unix.cc
+++ b/third_party/ijar/mapped_file_unix.cc
@@ -15,10 +15,11 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <stdio.h>
-#include <unistd.h>
 #include <sys/mman.h>
+#include <unistd.h>
 
 #include <algorithm>
+#include <limits>
 
 #include "third_party/ijar/mapped_file.h"
 
diff --git a/third_party/ijar/zlib_client.h b/third_party/ijar/zlib_client.h
index ed66163..0a917ff 100644
--- a/third_party/ijar/zlib_client.h
+++ b/third_party/ijar/zlib_client.h
@@ -17,6 +17,9 @@
 
 #include <limits.h>
 
+#include <limits>
+#include <stdexcept>
+
 #include "third_party/ijar/common.h"
 
 namespace devtools_ijar {
(END)
